support/ovscon2023/index.html

```diff
@@ -14,89 +14,148 @@
+<p>The solution introduces the following changes:</p>
+<ul><li>NB Schema changes to configure the DHCP Proxy for a Logical_Switch</li>
+    <li>A new pipeline stage in the Logical_Switch and the Logical_Router</li>
+    <li>New actions in the ovn-controller to proxy the DHCP requests and responses</li>
 </ul>
 
-<p>
-    Speakers will be notified of acceptance by November 3rd.
-</p>
-
-<p>
-    Topics that may be considered, among others, include:
-</p>
-
-<ul>
-    <li>The future of Open vSwitch and OVN (e.g., AF_XDP, P4, and eBPF).</li>
-
-    <li>NAT, DPI, and stateful processing with Open vSwitch.</li>
-
-    <li>Deploying and using OVN.</li>
-
-    <li>Testing and scaling OVN.</li>
-
-    <li>NIC acceleration of Open vSwitch.</li>
-
-    <li>Using Open vSwitch to realize NFV and service chaining.</li>
-
-    <li>Porting Open vSwitch to new operating systems, hypervisors, or container systems.</li>
-
-    <li>Integrating Open vSwitch into larger systems.</li>
-
-    <li>Troubleshooting and debugging Open vSwitch installations.</li>
-
-    <li>Open vSwitch development and testing practices.</li>
-
-    <li>Performance measurements or approaches to improving performance.</li>
-
-    <li>End-user or service provider experiences with Open vSwitch.</li>
-
-    <li>Hardware ports of Open vSwitch (existing, in progress, or speculative).</li>
-
-    <li>The relationship between OpenFlow and Open vSwitch.</li>
-
-    <li>Using, developing, or administering OpenFlow controllers in conjunction with Open vSwitch.</li>
-
-    <li>Comparisons to other implementations of features found in Open vSwitch (e.g. other OpenFlow implementations, other software switches, etc.).</li>
-
-    <li>Increasing the size and diversity of the Open vSwitch user and developer base.</li>
-
-    <li>Tutorials and walkthroughs of use cases.</li>
-
-    <li>Demos.</li>
-</ul>
+<p>Currently status of the feature: RFC is sent for review - <a href="https://www.mail-archive.com/ovs-dev@openvswitch.org/msg79088.html">https://www.mail-archive.com/ovs-dev@openvswitch.org/msg79088.html</a></p>
+
+<h3 id="a5">OVN expression parsing: Fighting inequality</h3>
+<p>Speaker: Ilya Maximets, Red Hat</p>
+<p>Having inequality matches in OVN ACLs was a big pain point for OVN users
+    for a long time as they could result in a huge number of OpenFlow rules
+    and very long processing times in ovn-controller.  This talk is an overview
+    of expression parsing in OVN's Logical Flows (including ACLs) as well as
+    some computer science, some math and a little bit of algorithms to bring
+    the number of OpenFlow rules down to reasonable.</p>
+
+<h3 id="a6">Ovs support userspace vxlan and geneve tunnel tso</h3>
```

Comment:
Maybe OVS here should be in all capitals?

Reply:
I agree, I'll update

Reply:
Pushed a correction for this.